kola/tests/ignition/filesystem.go

```diff
@@ -322,6 +323,14 @@ systemd:
 		Platforms:  []string{"qemu", "qemu-unpriv"},
 		MinVersion: semver.Version{Major: 3033},
 	})
+
+	register.Register(&register.Test{
+		Name:        "cl.ignition.extra_root_part",
```

Comment:
The naming sounds like a root partition gets created, how about something like `partition_on_boot_disk`?

Reply:
ok